test-clients/src/main/java/com/hedera/services/bdd/suites/perf/CryptoTransferLoadTest.java

```diff
@@ -117,7 +117,7 @@ protected HapiApiSpec runCryptoTransfers() {
 								.overridingProps(Map.of("hapi.throttling.buckets.fastOpBucket.capacity", "1300000.0")),
 						reduceFeeFor(CryptoTransfer, 2L, 3L, 3L),
 						cryptoCreate("sender")
-								.balance(initialBalance.getAsLong())
+								.balance((balance) -> settings.getInitialBalance())
```

Comment:
Any specific reason why a lambda here rather than just a function call?

Reply:
Fixed. Thanks @anighanta !

Reply:
I didn't mean to change it. We actually check the function first. Just wondering why

Reply:
Reverted to lambda as `balanceFn` sets the values in `HapiCryptoCreate.opBodyDef`